pyscriptjs/tests/unit/fakeruntime.ts

```diff
@@ -0,0 +1,40 @@
+import { Runtime } from "../../src/runtime"
+import type { PyodideInterface } from 'pyodide';
+
+export class FakeRuntime extends Runtime {
+
+    src: string;
+    name?: string;
+    lang?: string;
+    interpreter: PyodideInterface;
+    globals: any;
+
+    constructor() {
+        super(null);
+    }
+
+    async run(code: string) {
+        /* don't to anything */
```

Comment:
minor typo though: `don't to anything` --> `don't do anything`

Reply:
thanks, fixed by bfbaa38